### PR TITLE
CPO-46 Restructure Data Structures content

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

File: docs/modules/data-structures/pages/lock.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+[[monitoring-locks]]
+
+You can use the scripting feature of the Management Center to
+monitor the locks in your cluster. See the xref:scripting.adoc[Scripting section]
+to learn how to use this feature.
+
+You can use the below scripts to retrieve various information
+about the locks in your cluster.
+
+To find the number of active locks in your cluster, use the following script:
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/lock.adoc:11]: Can we add headings to these? For example: 'Finding Active Locks'

File: docs/modules/data-structures/pages/lock.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+[source,plain]
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/lock.adoc:13]:
```suggestion
[source,javascript]
```

File: docs/modules/data-structures/pages/lock.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    return "Active Lock Count : " + locks.size();
+
+}
+
+findLocks();
+----
+
+To print the locks in your cluster, use the following script:
+
+[source,plain]
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/lock.adoc:29]:
```suggestion
[source,javascript]
```

File: docs/modules/data-structures/pages/lock.adoc [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        }
+        return lockStr;
+}
+
+findLocks();
+----
+
+To force unlock a lock in your cluster, use the following script:
+
+[source,plain]
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/lock.adoc:48]:
```suggestion
[source,javascript]
```

File: docs/modules/data-structures/pages/lock.adoc [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    return 'OK';
+
+}
+
+forceUnlock('your_Lock_Name');
+----
+
+To check if a lock is being hold by a member, use the following script:
+
+[source,plain]
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/lock.adoc:62]:
```suggestion
[source,javascript]
```

File: docs/modules/data-structures/pages/lock.adoc [hunk 6]
```diff
@@ -0,0 +1,72 @@
+= Locks
+[[monitoring-locks]]
+
+You can use the scripting feature of the Management Center to
+monitor the locks in your cluster. See the xref:scripting.adoc[Scripting section]
+to learn how to use this feature.
+
+You can use the below scripts to retrieve various information
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/lock.adoc:8]:
```suggestion
You can use these JavaScript functions to retrieve various information
```

File: docs/modules/data-structures/pages/overview.adoc [hunk 7]
```diff
@@ -0,0 +1,24 @@
+= Overview
+:description: You can monitor the data structures in your cluster using the Storage, Compute, and Messaging menus in the Management Center.
+
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/overview.adoc:3]:
```suggestion
:page-aliases: monitor-imdg:monitor-dds.adoc
```

Redirects the old data structure page to this one.

**Review by JakeSCahill — CHANGES_REQUESTED** [review]
Looks good.  A couple of suggestions.

File: docs/modules/data-structures/pages/lock.adoc (hunk 1 above)
Comment (Serdaro) [reply docs/modules/data-structures/pages/lock.adoc:13]: added

**Pull request merged**